web/controllers/courseController/index.ts

```diff
@@ -2334,34 +2334,32 @@ async function groupMail(student: Student, courseId: number, subcourseId: number
 }
 
 /**
- * @api {POST} /course/:id/meeting/join JoinCourseMeeting
+ * @api {GET} /course/:id/subcourse/:subid/meeting GetCourseMeeting
```

Comment:
I think that is not consistent with that name https://github.com/corona-school/backend/blob/01abe7dc56d0d5d6b08785ed7018d5c33234bb8c/web/controllers/courseController/index.ts#L2360 and how it's accessible via the API, see this https://github.com/corona-school/backend/blob/01abe7dc56d0d5d6b08785ed7018d5c33234bb8c/web/index.ts#L139